lib/logger/lib/logger.ex

```diff
@@ -263,6 +279,30 @@ defmodule Logger do
   which must be set before the `:logger` application is started.
   Backends can also be added dynamically through `add_backend/2`.
 
+  For example, to add multiple backends to your application, modify your
+  `config.exs`:
```

Comment:
I would suggest we just say to modify the "configuration", without referring to a specific file name:

```suggestion
  configuration:
```

Reply:
I specifically wanted to mention the `config.exs` because some of use (cough) got confused with the other configuration functions in the `Logger` module (namely `configure/1` and `configure_backend/2`). 

How about saying :
```
For example, to add multiple backends to your application, modify your `config.exs` or its derivatives:
```

Reply:
> For example, to add multiple backends to your application, modify your `config/config.exs` or its derivatives: